cmd/autoheal/healer.go

```diff
@@ -113,6 +122,20 @@ func (b *HealerBuilder) Build() (h *Healer, err error) {
 	h.k8sClient = b.k8sClient
 	h.config = config
 
+	// Create the actions memory:
+	// if Throttling is "" or 0h then ignore throttling altogether.
+	if config.Throttling().Interval() != 0*time.Second {
```

Comment:
You say it is "" or 0h, but it will never be "" as it isn't an string, at least not here.

Why `0*time.Second`? Isn't `0` enough?

Note also that I don't see the need for not creating the memory when throttling is disabled, it should work anyhow, and conditionally creating it or not complicates things here and in other places. I'd suggest to always create it, and make sure that it works as expected when the interval is zero.

Reply:
We can always create it and use the implementation as before the only issue is in that case we would add items to the memory and the purge them right away on the next `Has` / run of purge worker - this whole behavior is not needed, but if you think we can stand to bare carrying these items in memory than I will revert to my latest commit.

Reply:
As for you're first comment you are absolutely correct :) :+1:

Reply:
Yes, I think that disabling throttling is only interesting for development environments, and even in that case we will want to have throttling, maybe with a shorter (but not zero) interval.